packages/theming/src/utils/colors.ts

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import type { ChromaStatic } from 'chroma-js';
+
+// @ts-expect-error Ignoring missing type definition
```

Comment:
Given the status of `chroma-js` is `@ts-ignore` the better choice here? https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#ts-ignore-or-ts-expect-error

Reply:
Actually, it's preferable to add a `chroma-js.d.ts` under /utils/build and update the tsconfig.json accordingly.